src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -124,6 +124,11 @@
   {:name schema/Str
    :files [EnvironmentClassesFileEntry]})
 
+(def EnvironmentModulesInfo
+  "Schema for the return payload an environment_classes request."
+  {:name schema/Str
+   :modules schema/Any}) ; TODO: fixme
```

Comment:
Cool, I think this approach makes sense to me, especially since it's already being used for `environment_classes`. I'll go ahead and rename the function too.